test/helpers.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var _ = require('lodash');
+var cluster = require('cluster');
+
+exports.workerCount = workerCount; // FIXME(sam) use live workers instead?
+exports.liveWorkers = liveWorkers;
+exports.pickWorker = pickWorker;
+exports.assertWorker = assertWorker;
+
+function liveWorkerIds() {
+  var workerIds = Object.keys(cluster.workers).filter(function(id) {
+    var worker = cluster.workers[id];
+    var isAlive = worker.process.signalCode == null &&
+      worker.process.exitCode == null;
+    return isAlive;
+  });
+  return workerIds.sort(function(a, b) {
```

Comment:
This aspect of the behaviour isn't needed here except to turn up the volume on the code smell of reimplementing features in tests.

Perhaps if this `liveWorkerIds` is useful in multiple contexts it should be extracted to a separate module (node module, not npm module)?